core/object/script_language.cpp

```diff
@@ -460,6 +459,51 @@ void ScriptLanguage::get_core_type_words(List<String> *p_core_type_words) const
 
 void ScriptLanguage::frame() {
 }
+TypedArray<int> ScriptLanguage::CodeCompletionOption::get_option_characteristics(String base) {
```

Comment:
Parameters should use `p_` prefix, and I guess this one could be 
```suggestion
TypedArray<int> ScriptLanguage::CodeCompletionOption::get_option_characteristics(const String &p_base) {
```